astroquery/heasarc/core.py

```diff
@@ -171,7 +171,7 @@ def query_region_async(self, position: Union[coordinates.SkyCoord, str],
         # Generate the request
         request_payload = self._args_to_payload(
             mission=mission,
-            entry="{},{}".format(c.ra.degree, c.dec.degree),
+            entry="{:.10f},{:.10f}".format(c.ra.degree, c.dec.degree),
```

Comment:
just an aside, this is in PR #2347 - so that should be merged first, then this will disappear from the diff

Reply:
that's right. I meant this one to be merged after #2347 .